### DISCUSSION TITLE
Open Dialog via Main Menu

### DISCUSSION THREAD

**Discussion by axelcasauran** [serenity-is/Serenity#6268] [Help] [answered]
Hi,

How can I open the dialog via navigation / main menu without accessing or showing the grid?

Thanks.

![2022-02-14 08_37_15-Window](https://user-images.githubusercontent.com/13443821/153782806-93ceff98-690d-4cc1-9d71-f751e361222e.png)

**Comment by wezmag** [comment, marked as answer]
There is a sample in Demo site.
URL: https://serenity.is/demo/BasicSamples/EntityDialogAsPanel/
Source code: 
1. Navigation: https://github.com/serenity-is/common-features/blob/63780f49774d11bb3755807991aa9ff96d44b9d6/src/Serenity.Demo.BasicSamples/Serenity.Demo.BasicSamples/Dialogs/BasicSamplesNavigation.Dialogs.cs#L10
2. Dialog: https://github.com/serenity-is/common-features/blob/master/src/Serenity.Demo.BasicSamples/Serenity.Demo.BasicSamples/Dialogs/EntityDialogAsPanel/EntityDialogAsPanel.ts

BTW, the URL in the navigation has ID parameter (11077), you can remove the ID to display a CREATE dialog.

**Reply by axelcasauran** [reply]
Thank you.. It works.
